code/obj/window.dm

```diff
@@ -1064,6 +1065,19 @@
 /obj/window/auto/feather/proc/repair()
 	src.health = min(src.health + 10, src.health_max)
 
+/obj/window/auto/feather/Crossed(atom/movable/mover)
```

Comment:
Crossed() or Bumped()?

Reply:
Crossed in this case. Not entirely sure why, but if Cross evaluates to true, Bumped is never triggered. I don't need to do that fir the walls because I modified the pathing algo, and they're turfs, but windows are objects on turfs so Cross/ed it is